compiler/AST/TryStmt.cpp

```diff
@@ -18,20 +18,38 @@
  */
 
 #include "TryStmt.h"
+
 #include "AstVisitor.h"
+#include "CatchStmt.h"
+
+BlockStmt* TryStmt::build(bool tryBang, Expr* expr) {
+  BlockStmt* body = new BlockStmt(expr);
+  return buildChplStmt(new TryStmt(tryBang, body, NULL));
+}
 
 BlockStmt* TryStmt::build(bool tryBang, BlockStmt* body) {
-  return buildChplStmt(new TryStmt(tryBang, body));
+  return buildChplStmt(new TryStmt(tryBang, body, NULL));
+}
+
+BlockStmt* TryStmt::build(bool tryBang, BlockStmt* body, BlockStmt* catches) {
+  return buildChplStmt(new TryStmt(tryBang, body, catches));
 }
 
 BlockStmt* TryStmt::buildChplStmt(Expr* expr) {
   return new BlockStmt(expr, BLOCK_SCOPELESS);
 }
 
-TryStmt::TryStmt(bool tryBang, BlockStmt* body) : Stmt(E_TryStmt) {
+TryStmt::TryStmt(bool tryBang, BlockStmt* body, BlockStmt* catches) : Stmt(E_TryStmt) {
```

Comment:
It would be nice to have a comment here to indicate that catches are stored in a BlockStmt to make parsing more convenient.